docs/how-to/sync-users.mdx

```diff
@@ -174,8 +174,18 @@ The rest of the schema for the call consists of the following parameters:
 
 - **`attributes`** (Optional) object (Attributes)
   Default: {}
-  User attributes are used to enforce attribute-based access control policies. You can read more about
-  [how to define these attributes here](/how-to/build-policies/abac/defining-attributes).
+    User attributes are used to enforce attribute-based access control policies. You can read more about
+    [how to define these attributes here](/how-to/build-policies/abac/defining-attributes).
+
+- **`role_assignments`** (Optional) array (Role Assignments),
+    Default: None,
+    An array of role assignments to assign to the user. Each role assignment is an object with the following properties:
+    - **`role`** string (Role) The role key or ID.
+    - **`tenant`** (Optional) string (Tenant) The tenant key or ID.
+    - **`resource_instance`** (Optional) string (Resource Instance) The resource instance key or ID.
```

Comment:
Maybe in the parathness of optional of the resource instance do like (Optional - see note below). it little bit confusing cause it not really optional field